plugin-server/src/cdp/consumers/cdp-cyclotron-segment-worker.test.ts

```diff
@@ -0,0 +1,1602 @@
+                    $initial_geoip_subdivision_1_code: 'NSW',
+                    $initial_geoip_subdivision_1_name: 'New South Wales',
+                    $initial_geoip_subdivision_2_code: null,
+                    $initial_geoip_subdivision_2_name: null,
+                    current_organization_membership_level: 15,
+                },
+                groups: {},
+                app_version: null,
+                platform: 'Desktop',
+                os_name: 'Mac OS X',
+                os_version: '10.15.7',
+                device_brand: '',
+                device_manufacturer: null,
+                device_model: null,
+                carrier: '',
+                country: 'Australia',
+                region: '',
+                city: 'Sydney',
+                language: null,
+                userAgent:
+                    'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/135.0.0.0 Safari/537.36',
+                userAgentParsing: true,
+                includeRawUserAgent: false,
+                utm_properties: {
+                    utm_term: null,
+                    utm_medium: null,
+                    utm_source: null,
+                    utm_content: null,
+                    utm_campaign: null,
+                },
+                referrer:
+                    'http://localhost:8000/project/1/pipeline/new/destination/hog-template-meta-ads?showPaused=true&kind&search=meta',
+                library: 'web',
+                userAgentData: {
+                    model: '',
+                    platformVersion: '',
+                },
+                internal_partner_action: 'logEventV2',
+            }
+
+            fn = await insertHogFunction({
+                name: 'Plugin test',
+                template_id: 'segment-amplitude',
+            })
+
+            const invocation = createSegmentInvocation(fn, amplitudeInputs)
+
+            mockFetch.mockResolvedValue({
+                status: 200,
+                json: () => Promise.resolve({ total_count: 1 }),
+                text: () =>
+                    Promise.resolve(
+                        JSON.stringify({
+                            code: 200,
+                            server_upload_time: 1747910402315,
+                            payload_size_bytes: 22207,
+                            events_ingested: 1,
+                        })
+                    ),
+                headers: {},
+            })
+
+            const { invocationResults } = await processor.processBatch([invocation])
+
+            expect(invocationResults.length).toBe(1)
+
+            invocationResults[0].logs.forEach((x) => {
+                if (typeof x.message === 'string' && x.message.includes('Function completed in')) {
+                    x.message = 'Function completed in [REPLACED]'
+                }
+            })
+
+            expect(invocationResults[0].logs).toMatchInlineSnapshot(`
```

Comment:
When you have really big snapshots like this we should just use `toMatchSnapshot` - inline ones are good when there is a limited number of options. Now the file is so huge its virtually unreadable.